File: src/main/java/cn/lnkdoc/sdk/uia/instance/bjtoon/util/UiaSigner.java
package cn.lnkdoc.sdk.uia.instance.bjtoon.util;

/**
 * Sign Interface
 *
 * @author qinyi
 * @version v 1.0 2018-10-22
 */
@SuppressWarnings(value = {"unused"})
public interface UiaSigner {
	
	/**
	 * sign
	 * 
	 * @param sourceContent content to be signed
	 * @param signType signType
	 * @param charset charset
	 * @return String
	 */
	String sign(Object sourceContent, String signType, String charset);

}

File: src/main/kotlin/cn/lnkdoc/sdk/uia/instance/bjtoon/util/UiaSigner.kt
package cn.lnkdoc.sdk.uia.instance.bjtoon.util

/**
 * Sign Interface
 *
 * @author qinyi
 * @version v 1.0 2018-10-22
 */
@Suppress("unused")
interface UiaSigner {
    /**
     * sign
     *
     * @param sourceContent content to be signed
     * @param signType signType
     * @param charset charset
     * @return String
     */
    fun sign(sourceContent: Any?, signType: String?, charset: String?): String?
}

